src/server.c

```diff
@@ -235,43 +235,41 @@ void serverServe(Server *server)
 
     int sock = makeSocket(server->port);
     int newSock;
+    int fd_size = sock + 1;
 
-    socklen_t size;
+    struct sockaddr_in addr;
+
```

Comment:
Is that means  there shouldn't  be a newline between `struct sockaddr_in addr;` and  `socklen_t size = sizeof(addr);` ?